core/templates/domain/state_card/state-card.model.ts

```diff
@@ -223,7 +245,6 @@ export class StateCard {
     return lastInputResponsePair.learnerInput;
   }
 
-
```

Comment:
Please undo this.

Reply:
All other member functions have a single empty line between them so it's more consistent with the rest of the file. Although this change is not related to this PR so I can undo it if that's the reason

Reply:
Ah, thanks for the explanation. Let's keep it in that case, to make things consistent. :)